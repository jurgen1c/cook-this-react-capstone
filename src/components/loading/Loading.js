const Loading = () => (
  <div>
    <img src="/images/loading.gif" alt="Loading Gif" />
  </div>
);

export default Loading;
