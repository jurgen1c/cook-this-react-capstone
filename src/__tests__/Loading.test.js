import React from 'react';
import renderer from 'react-test-renderer';
import Loading from '../components/loading/Loading';

test('Componenet renders correctly', () => {
  const component = renderer.create(
    <Loading />,
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});