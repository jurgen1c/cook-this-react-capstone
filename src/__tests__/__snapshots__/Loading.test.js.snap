// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Componenet renders correctly 1`] = `
<div
  className="loader-cont"
>
  <img
    alt="Loading Gif"
    className="loader"
    src="/images/loading.gif"
  />
</div>
`;
